Gemfile.lock

```diff
@@ -1,354 +0,0 @@
-GIT
```

Comment:
I thought this already disappeared in the mongo PR but I guess your forgot doing it there

Reply:
You are right. I had recovered `Gemfile.lock` before merged it. By bad.